docker/images/dev-cli/Dockerfile

```diff
@@ -0,0 +1,59 @@
+# Creates an image for skycoin development
+FROM golang:1.10.0-stretch
+
+# Installs software
+RUN apt-get update && apt-get install -y --no-install-recommends \
+        cmake \
+        libpcre3-dev \
+        gdbserver \
+        gdb \
+        vim \
+        ctags \
+        vim-scripts \
+        screen \
+        sudo \
+        doxygen \
+    && apt-get clean \
+    && rm -rf /var/lib/apt/lists/* 
```

Comment:
@stdevGeekmidget in here install [Graphviz](https://www.graphviz.org/) , and [pdflatex](www.math.rug.nl/~trentelman/jacob/pdflatex/pdflatex.html)